### ISSUE TITLE
v3 has bad dependencies

### ISSUE THREAD

**Issue by JosefJezek** [vaadin/vaadin-grid#1081]
https://github.com/vaadin/vaadin-grid/blob/v3.0.2/bower.json#L49

Correct dependencies are
```
"iron-a11y-keys-behavior": "1 - 2",
 "polymer": "1.9 - 2",
...
```

**Comment by vicsstar** [comment]
@JosefJezek the dependencies have been updated.

**Comment by web-padawan** [comment]
Fixed in #1059 and released in v3.1.0
